homework-submission/hw-week1/src/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const staticList = [
```

Comment:
As far as I can see you are not using this `staticList` variable. The static list is hardcoded so I think you can remove this?

Reply:
Actually, I used it in staticComponent in my file line 71.